lighthouse-core/computed/trace-of-tab--deprecated.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Why not just put the warning message in `trace-of-tab.js` rather than create this weird deprecated file?

Reply:
yeah that's fair. I wanted a bit of extra attention drawn to not using it, but leaving `trace-of-tab` shim in place somewhat defeats that purpose, so I'll remove.